pywren/storage/s3_backend.py

```diff
@@ -24,7 +41,7 @@ def put_object(self, key, data):
         """
         self.s3client.put_object(Bucket=self.s3_bucket, Key=key, Body=data)
 
-    def get_object(self, key):
+    def get_object(self, key, data_byte_range=None):
```

Comment:
update the documentation below on what `data_byte_range` is and what its format should be ?